test/server/models/Collective.test.js

```diff
@@ -480,16 +483,30 @@ describe('server/models/Collective', () => {
     expect(collective.image).to.equal('https://www.gravatar.com/avatar/a97d0fcd96579015da610aa284f8d8df?default=404');
   });
 
-  it('computes the balance ', () =>
-    collective.getBalance().then(balance => {
+  it('computes the balance (v1)', () =>
+    collective.getBalance({ version: 'v1' }).then(balance => {
       let sum = 0;
       transactions.map(t => (sum += t.netAmountInCollectiveCurrency));
       expect(balance).to.equal(sum);
     }));
 
-  it('computes the balance until a certain month', done => {
+  it('computes the balance (v2 - default)', () =>
+    collective.getBalance().then(balance => {
+      let sum = 0;
+      transactions.map(
+        t =>
+          (sum +=
+            (t.amountInHostCurrency || 0) +
+            (t.hostFeeInHostCurrency || 0) +
+            (t.platformFeeInHostCurrency || 0) +
+            (t.paymentProcessorFeeInHostCurrency || 0)),
+      );
+      expect(balance).to.equal(sum);
```

Comment:
I often have a preference for hardcoding values in that kind of situation, otherwise there's always a risk of reproducing in the tests the same error that's in the code. But this one is fairly simple, I don't have a strong opinion.